src/utilities/encodedImageStorage/uploadEncodedImage.ts

```diff
@@ -3,17 +3,37 @@ import * as fs from "fs";
 import { writeFile } from "fs/promises";
 import { encodedImageExtentionCheck } from "./encodedImageExtensionCheck";
 import { errors, requestContext } from "../../libraries";
-import { INVALID_FILE_TYPE } from "../../constants";
+import { IMAGE_SIZE_LIMIT_KB, INVALID_FILE_TYPE } from "../../constants";
 import { EncodedImage } from "../../models/EncodedImage";
 import path from "path";
 import { deletePreviousImage } from "./deletePreviousImage";
 
+const checkImageSizeLimit = (size: number): boolean => {
+  return size > 0 && size <= 20000;
+};
+
 export const uploadEncodedImage = async (
   encodedImageURL: string,
   previousImagePath?: string | null
 ): Promise<string> => {
   const isURLValidImage = encodedImageExtentionCheck(encodedImageURL);
 
+  const data = encodedImageURL.replace(/^data:image\/\w+;base64,/, "");
+  const stringLength = data.length;
+  const sizeInKb =
+    (4 * Math.ceil(stringLength / 3) * 0.5624896334383812) / 1000;
```

Comment:
Add comments to explain this calculation. If there are any constants, like `0.5624896334383812`, declare then as a const with the appropriate name, and then use the same.